core/src/main/java/org/apache/iceberg/variants/PrimitiveWrapper.java

```diff
@@ -210,4 +210,9 @@ public int writeTo(ByteBuffer outBuffer, int offset) {
 
     throw new UnsupportedOperationException("Unsupported primitive type: " + type());
   }
+
+  @Override
+  public String toString() {
```

Comment:
Can we add basic test coverage for the changes?

Reply:
These are for convenience and are not part of a contract. I don't think it is worth adding tests.